client/apps/edit/components/content2/section_controls/test/index.coffee

```diff
@@ -1,244 +0,0 @@
-benv = require 'benv'
```

Comment:
this is a duplicate of index.test.coffee, somehow the old one got back in after rename.